### PR TITLE
Cleaned cache, reinstalled dependencies

### LINKED ISSUE #77: cloudbuild errors
As a developer, I want the pull requests to pass all CloudBuild checks.

### PR DESCRIPTION
**Pull request by veikkahonkanen**
Resolves #77

### CODE REVIEW

File: src/functions/checks/checkGit.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { CheckErrorSet } from "../../interfaces/types";
  */
 export const checkGit = async () : Promise<CheckErrorSet> => {
 
-  let returnSet : CheckErrorSet = {check : "git", error : false, details : null};
+  const returnSet : CheckErrorSet = {check : "git", error : false, details : null};
```
Comment (eetupur) [inline src/functions/checks/checkGit.ts]: Does this really work with const, it's attrubutes are edited just below this.

Comment (jarinykanen) [inline src/functions/checks/checkGit.ts]:
```suggestion
  const returnSet : CheckErrorSet = {
    check : "git",
    error : false,
    details : null
  };
```

**Review by jarinykanen — APPROVED** [review]

File: src/interfaces/types.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export interface CheckErrorSet {
  */
 export interface CheckSet {
   checkable : string,
-  details : Object
+  details : unknown
```
Comment (eetupur) [inline src/interfaces/types.ts:35]: are we going with unknown and not <T>?

**Review by eetupur — COMMENTED** [review]
Found some concerning bits.

File: src/interfaces/types.ts (hunk 2 above)
Comment (veikkahonkanen) [reply src/interfaces/types.ts:35]: yes

File: src/functions/checks/checkGit.ts (hunk 1 above)
Comment (veikkahonkanen) [reply src/functions/checks/checkGit.ts]: yes

**Pull request merged**